backend/tests/routes/test_document_relationships.py

```diff
@@ -296,132 +296,3 @@ def test_get_relationship_documents_security(
 
     assert response_get.status_code == 404
     assert response_get.json() == {"detail": "Not Found"}
-
-
-# TODO : Un comment when relationships are going to replace associations
```

Comment:
Did this get replaced?

Reply:
Ah yes - I moved this here - but instead now I've  updated the test in `test_documents.py` to validate this when getting the document details. This tests is a bit of a mother-of-all tests and probably needs breaking down but not today :grin: